### PR TITLE
limesuite: 18.06.0->18.10.0

### PR DESCRIPTION
**Pull request by markuskowa**
###### Motivation for this change
Update and do some cleanup.

###### Things done

* update
* remove example binaries from  output
* enable parallel building

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/applications/misc/limesuite/default.nix [hunk 1]
```diff
@@ -36,10 +38,6 @@ in stdenv.mkDerivation {
 
     mkdir -p $out/share/limesuite
     cp bin/Release/lms7suite_mcu/* $out/share/limesuite
-
-    cp bin/dualRXTX $out/bin
-    cp bin/basicRX $out/bin
-    cp bin/singleRX $out/bin
```
Comment (timokau) [inline pkgs/applications/misc/limesuite/default.nix]: Why did you remove these?

Comment (markuskowa) [reply pkgs/applications/misc/limesuite/default.nix]: Those are compiled from example code. The binaries are not useful by themselves (I did not realize that in the first version). For example `basicRX` just sets the receiver to 800MHz and discards the received samples.

**Comment by timokau** [comment]
Looks good, thanks!
